src/support/z_context.erl

```diff
@@ -416,10 +417,12 @@ abs_url(Url, Context) ->
 
 
 %% @doc Fetch the pid of the database worker pool for this site
+-spec db_pool(#context{}) -> atom().
```

Comment:
The spec in line 420 uses a record, please define a type for the record and use that instead.